source/components/GlobalFooter/components/BottomBar.js

```diff
@@ -1,9 +1,11 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
 import { useTranslation } from 'react-i18next';
 
+import I18nNamespaceContext from '../context/I18nNamespaceContext';
+
 const BottomBar = ({ model }) => {
-    const [t] = useTranslation();
+    const [t] = useTranslation(useContext(I18nNamespaceContext));
```

Comment:
how about extracting it to the separate hook?

Reply:
will do! but as a separate PR